Fix service script for ubuntu

fatihky opened the pull request:
see comments in the changes

fatihky commented on daemon_linux_systemv.go:
```diff
@@ -235,23 +235,29 @@ lockfile="/var/lock/subsys/$proc"
         echo $! > $pidfile
         touch $lockfile
-        success
+        echo "service $servname started"
```
I couldn't find where the `success` is, so I removed it with simple message.

fatihky commented on daemon_linux_systemv.go:
```diff
@@ -235,23 +235,29 @@ lockfile="/var/lock/subsys/$proc"
+            rm $pidfile
+        fi
+    fi
+
```
Pid file existence check is smart. But that files don't always contain valid pids. So I have added code that checks if pid is valid(running) or not. And if not running, it deletes the pid file.

fatihky commented on daemon_linux_systemv.go:
```diff
@@ -235,23 +235,29 @@ lockfile="/var/lock/subsys/$proc"
     else
-        failure
+        # failure
         echo
```
I couldn't find where the `failure` is, so I simply commented it out.

takama commented:
Thank you Fatih,
I will check it on weekend. Sorry for delay, too much work.
Thank you.

fatihky commented on daemon_windows.go:
```diff
@@ -6,10 +6,12 @@
 )
 
-var ErrWindowsUnsupported = errors.New("windows daemon is not supported")
+var ErrWindowsUnsupported = errors.New("Adding as a service failed. Download and place nssm.exe in the path to install this service as an service. NSSM url: https://nssm.cc/")
```
I have tried adding executable as a service with `sc.exe`, but it didn't work. Then I have tried `nssm`, and it worked like a charm. So I have added windows support with `nssm`. I have to try it now at in windows.

fatihky commented on daemon_windows.go:
```diff
@@ -27,37 +29,66 @@ func newDaemon(name, description string, dependencies []string) (Daemon, error)
+	}
+
+	return installAction + " completed.", nil
 }
```
This isn't the best way. But I think it will work. I am not sure about `Descriptilon` parameter. I have to ry it after switching to windows.

fatihky commented:
Sorry I was installed restyaboard in daemon's directory. So I have commited some files unnecessary(installation bash script and log file). I have deleted them at last commit. If you want, delete this pr and copy files from my repo by hand.

I have fixed service installation for ubuntu and windows(I have tried install and remove, they are working.).

takama commented:
Thank you Fatih,
Right now we have two different solutions under one pull request. Could you please split the pull request by on two different pull requests?

fatihky commented:
@takama I should close this pr and make two different prs, right? If yes, I will do in my next freetime(probably in today or tomorrow).

takama commented:
Yes, I will be very grateful to you for it.
